blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/inputs/handlers/HttpHandlerIntegrationTest.java

```diff
@@ -151,6 +150,42 @@ public void testHttpIngestionPartialInvalidCollectionTime() throws Exception {
     }
 
     @Test
+    public void testHttpIngestionAllInvalidStringBooleanMetrics() throws Exception {
```

Comment:
@usnavi @ChandraAddala , FYI, I added 4 more HTTP integration tests:
* post to /ingest all metrics with wrong types (should get 400)
* post to /ingest some metrics invalid, some valid (should get 207)
* post to /ingest/multi all metrics with wrong types (should get 400)
* post to /ingest/multi some metrics invalid, some valid (should get 207)